display-guild.lic

```diff
@@ -0,0 +1,86 @@
+Contacts = YAML.load_file('./scripts/data/base-contacts.yaml')
+Contacts = Contacts['contacts']
+
+#DEBUG INFO
+#puts "---Base Contacts---"
+#puts Contacts.inspect
+#puts Contacts.class
+
+#Check for <CharName>-contacts.yaml file and merges it if exists
+if File.exist?('./scripts/profiles/'+checkname+'-contacts.yaml')
+	puts "Personal Contact file found! Now importing!"
+	
+	Personal_Contacts = YAML.load_file('./scripts/profiles/'+checkname+'-contacts.yaml')
+	Personal_Contacts = Personal_Contacts['contacts']
+
+	#DEBUG INFO
+	#puts "---Personal Contacts---"
+	#puts Personal_Contacts.inspect
+	#puts Personal_Contacts.class
+
+	Contacts.merge!(Personal_Contacts)
+	Names = Contacts.map {|row| row[0]}
+	puts "***Successfully loaded " + Names.length.to_s + " contacts."
+	
+	#DEBUG INFO
+	#puts "---Names Array---"
+	#puts Names.inspect
+	#puts Names.class
+end
+
+#DEBUG INFO
+#puts "---Merged Contacts---"
+#puts Contacts.inspect
+#puts Contacts.class
+#puts "-----"
+
+
+# quiet
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#stalker

```

Comment:
Stalker was the original name I was using for this script, but decided to change it to something more descriptive. I figured there wasn't much need to go back and change the name in the code itself, but I had forgotten about the reference to the documentation. That will need to be changed.